src/App.css

```diff
@@ -78,4 +88,11 @@ button.fc-button.finos-calendar-event-details-close {
     max-height: 1em;
     margin: 1em 0;
   }
+}
+
+@media only screen and (prefers-color-scheme : dark){
+  #root, .main, .finos-calendar, .finos-calendar-event-details{
+    background-color: #222;
```

Comment:
Looking at the links, probably we should recolor <a> tag to be less of blue color contrast, something less bright.

Reply:
I also think the same. Let's pick a few colors and I'll test them one by one, share screenshots here and we can come to a conclusion. Can you suggest any color?

Reply:
I would guess #343 would be a good match - dark greenish, matches the yellowish hue of the current day

Reply:
![Screenshot (351)](https://github.com/finos/calendar/assets/75676784/ed3b58ad-4374-4b66-b608-5e838f947513)
How's this?

Reply:
I like it, @maoo, what you think?